src/game/game_main.cpp

```diff
@@ -48,24 +85,65 @@ void show_int(char *buffer, void *info) {
 
 void entity_registration() {
     REGISTER_TYPE(std::vector<int>, show_buffer);
-
-    REGISTER_ENTITY(A);
-    REGISTER_ENTITY(MyEnt);
 }
 
+std::vector<Paddle *> paddles = {};
```

Comment:
You don't want a list of pointers. This causes you to access uninitialized memory. More comments further down.